tests/dragonfly/snapshot_test.py

```diff
@@ -80,4 +80,5 @@ async def test_snapshot(self, df_seeder_factory, df_server):
 
         time.sleep(60)
 
-        assert (self.tmp_dir / "test-periodic.rdb").exists()
+        files = [f for f in os.listdir(self.tmp_dir) if f.startswith('test-periodic')]
+        assert len(files) > 0 and any(f.endswith('summary.dfs') for f in files)
```

Comment:
We have a bug here, that even with the extension, the timestamp is still included

But its not connected with making the pytest run